packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -218,6 +218,7 @@ function Preview({
 }: Props) {
   const currentMousePosition = useRef<MouseEvent<HTMLDivElement>>();
   const wrapperDivRef = useRef<HTMLDivElement>(null);
+  const [ignoreMouseUp, setIgnoreMouseUp] = useState(false);
```

Comment:
I wonder if we need another state for that. I imagine that we ignore right-click all together in terms of sending touch events. When inspector is on we again ignore all touches until it get's disabled.